app/controllers/auth/registrations_controller.rb

```diff
@@ -142,4 +142,12 @@ def require_rules_acceptance!
   def set_cache_headers
     response.cache_control.replace(private: true, no_store: true)
   end
+
+  def is_flashing_format? # rubocop:disable Naming/PredicateName
```

Comment:
This is fine as-is ... but curious how "permanent" this removal is? If very permanent (and if they are not used elsewhere?) ... would removing the i18n values, and/or setting them to `''` have same effect?

Reply:
I'd say very permanent. Good call about removing the strings, I wonder why CI isn't complaining about that right now...

Reply:
Because all `devise` keys are ignored in i18n-tasks configuration.

Reply:
> I'd say very permanent. Good call about removing the strings

With the i18n strings removed - is this `is_flashing_format?` override still needed? (I'm not sure if devise behavior is to ignore missing and not populate flash, or put in some sort of missing message or default string or something else...)

> Because all `devise` keys are ignored in i18n-tasks configuration.

Unrelated to this PR, a periodic audit of "are we ignoring things we dont need to ignore anymore" would be useful. Probably could not be automated, though.